packages/cache/src/internal/config.ts

```diff
@@ -0,0 +1,24 @@
+export function getRuntimeToken(): string {
+  const token = process.env['ACTIONS_RUNTIME_TOKEN']
+  if (!token) {
+    throw new Error('Unable to get the ACTIONS_RUNTIME_TOKEN env variable')
+  }
+  return token
+}
+
+// TODO: Use the feature flag to determine the cache service version
+export function getCacheServiceVersion(): string {
+  return process.env['ACTIONS_CACHE_SERVICE_VERSION'] || 'v1'
+}
```

Comment:
when this is fixed, we should include some logic to check for results env too. we don't want to end up removing the feature flag after ship and attempting to hit v2 in GHES